packages/diagrams/frontend/sirius-components-diagrams/src/renderer/node/FreeFormNode.tsx

```diff
@@ -137,9 +142,8 @@ export const FreeFormNode = memo(({ data, id, selected, dragging }: NodeProps<Fr
         onDragOver={onDragOver}
         onDrop={handleOnDrop}
         data-testid={`FreeForm - ${data?.targetObjectLabel}`}>
-        <div style={{ ...backgroundStyle }}>
-          {data.insideLabel && <Label diagramElementId={id} label={data.insideLabel} faded={data.faded} transform="" />}
-        </div>
+        <div style={{ ...backgroundStyle }} />
```

Comment:
Is this normal that this div is now empty?

Reply:
Yes, this div only displays the background